src/main/java/edu/ucsb/cs156/rec/entities/RecommendationRequest.java

```diff
@@ -0,0 +1,33 @@
+package edu.ucsb.cs156.rec.entities; 
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "recommendationrequest")
+public class RecommendationRequest {
+    private String requesterEmail;
```

Comment:
It might be better to just store the userId field which can then be used to look up the name/email.  But I'll let that go for right now; that can be something you explore down the road.